src/app/clusters/scenes-server/SceneTableImpl.h

```diff
@@ -31,6 +31,7 @@ static constexpr uint16_t kMaxScenesPerFabric   = CHIP_CONFIG_SCENES_MAX_PER_FAB
 static constexpr uint16_t kMaxScenesPerEndpoint = CHIP_CONFIG_MAX_SCENES_PER_ENDPOINT;
 
 using clusterId = chip::ClusterId;
+using namespace app::Clusters::Scenes::Structs;
```

Comment:
Please do not put `using` of any sort at toplevel in headers.  Doing that within class scope is fine, of course.